ChaosMod/Memory/Hooks/DrawingHooks.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace Hooks
+{
+	void DrawLine(float x1, float y1, float x2, float y2, float width, int r, int g, int b, int a);
+
+	inline void DrawLine(rage::Vector2 firstPos, rage::Vector2 secondPos, Color color, float thickness)
```

Comment:
Should also include the headers containing the types (or a forward declaration if you desire).